src/modules/register/Register.jsx

```diff
@@ -233,17 +226,12 @@ const Form = styled.form`
 	}
 `;
 
-function Input(props) {
+function Input({type, name, autoComplete, onChange, value, className = '', children}) {
 	return (
-		<div className={'input-container' + props.class}>
-			<input
-				onChange={props.onChange}
-				type={props.type}
-				name={props.name}
-				id={props.name}
-				autoComplete={props.autoComplete}
-			/>
-			<label htmlFor={props.name}>{props.name}</label>
+		<div className={`input-container ${className}`}>
+			<input autoFocus onChange={onChange} type={type} name={name} id={name} autoComplete={autoComplete} value={value} />
+			<label htmlFor={name}>{name}</label>
+			{children}
```

Comment:
Here is the `children` prop, and how we get `CheckBoxWrapper` in here for password.

[`children`](https://reactjs.org/docs/jsx-in-depth.html#children-in-jsx) is a weird prop, because you can either pass it in by nesting it:
```jsx
<Input>
  <CheckboxWrapper />
</Input>
```
or by passing it in as a prop, as we are doing:
```jsx
<Input children={<CheckboxWrapper />} />
```